python/sparkdl/transformers/keras_applications.py

```diff
@@ -70,15 +73,14 @@ def model(self, preprocessed, featurize):
     def inputShape(self):
         pass
 
-    @abstractmethod
-    def numOutputFeatures(self):
+    def testPreprocess(self, inputImage):
         """
-        For testing
+        For testing, the keras preprocess function for the model.
         """
-        pass
+        return self.preprocess(inputImage)
 
     @abstractmethod
-    def testKerasModel(self):
+    def testKerasModel(self, include_top):
```

Comment:
How do you feel about making these test methods "private"? I think we should make it clear they are not part of the public API.